starknet_devnet/server.py

```diff
@@ -310,26 +41,26 @@ def main():
 
     if args.load_path:
         try:
-            starknet_wrapper = StarknetWrapper.load(args.load_path)
+            state.load(args.load_path)
         except (FileNotFoundError, pickle.UnpicklingError):
             sys.exit(f"Error: Cannot load from {args.load_path}. Make sure the file exists and contains a Devnet dump.")
 
-    dumper.dump_path = args.dump_path
-    dumper.dump_on = args.dump_on
+    state.dumper.dump_path = args.dump_path
+    state.dumper.dump_on = args.dump_on
 
     if args.lite_mode:
-        starknet_wrapper.lite_mode_block_hash = True
-        starknet_wrapper.lite_mode_deploy_hash = True
+        state.starknet_wrapper.lite_mode_block_hash = True
+        state.starknet_wrapper.lite_mode_deploy_hash = True
     else:
-        starknet_wrapper.lite_mode_block_hash = args.lite_mode_block_hash
-        starknet_wrapper.lite_mode_deploy_hash = args.lite_mode_deploy_hash
+        state.starknet_wrapper.lite_mode_block_hash = args.lite_mode_block_hash
+        state.starknet_wrapper.lite_mode_deploy_hash = args.lite_mode_deploy_hash
 
     try:
         meinheld.listen((args.host, args.port))
         meinheld.run(app)
     finally:
         if args.dump_on == DumpOn.EXIT:
-            dumper.dump(dumper.dump_path)
+            state.dumper.dump(state.dumper.dump_path)
```

Comment:
Not related to this PR, but could we not pass `state.dumper.dump_path` here?

Reply:
[d1b2a89](https://github.com/Shard-Labs/starknet-devnet/pull/64/commits/d1b2a894948e4089fffefbaffa26497404fe7e00)